pallets/phala/src/compute/poolproxy.rs

```diff
@@ -26,22 +27,31 @@ pub struct StakePool<AccountId, Balance> {
 	pub workers: VecDeque<WorkerPublicKey>,
 	/// The workers in cd in the pool
 	pub cd_workers: VecDeque<WorkerPublicKey>,
-
+	/// The account generated to store ppha locked in miners with its asset account and controlled by the pallet
 	pub lock_account: AccountId,
-
+	/// The account generated to maintain owner rewards with its asset account and controlled by the pallet
 	pub owner_reward_account: AccountId,
 }
 
 #[derive(Encode, Decode, TypeInfo, Clone, PartialEq, Eq, RuntimeDebug)]
 pub struct Vault<AccountId, Balance> {
+	/// General attributes of a pool
 	pub basepool: basepool::BasePool<AccountId, Balance>,
-
+	/// Record the share price when function: `maybe_gain_owner_shares` is called last time
 	pub last_share_price_checkpoint: Balance,
-
+	/// The commission the owner takes
+	///
+	/// The caculate formula of additional shares minted for pool owner is present below:
+	/// additional_shares = total_value / (current_price - commission * min(current_price - last_share_price_checkpoint, 0)) - total_shares
+	/// When pool's share price is lower than last_share_price_checkpoint, the pool is regarded as in deficit and won't gain any addtional owner shares
+	/// The commission is actually indicate the percent of profit that earned by the pool should be redistributed to the pool owner
```

Comment:
```suggestion
	/// The calculated formula of the additional shares minted for pool owner is present below:
	///
	/// ```
	/// additional_shares = total_value / (current_price - commission * min(current_price - last_share_price_checkpoint, 0)) - total_shares
	/// ```
	///
	/// When the pool's share price is lower than `last_share_price_checkpoint`, the pool made no profit  since the last checkpoint. So it won't gain any additional owner shares.
	///
	/// The `commission` indicates the percentage of the equivalent profits earned by the pool should be redistributed to the pool owner in the form of shares.
```

Reply:
Better to grammarly it. I don't speak good English

Reply:
done